apps/AmplifyDemo/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
You may target a later version, e.g., [ES2015 in the JS demo](https://github.com/aws/amazon-chime-sdk-js/blob/2b133458cfae7ff45aeb942bb8e2fc19b23534cd/config/tsconfig.base.json#L17), or [even later one like React demo](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/dfe904fec85b5f7c21af690e37c32e4806320e81/demo/meeting/tsconfig.json#L10)